installer/datafiles/linux.data

```diff
@@ -119,6 +119,13 @@ if [ -d "$OMS_CERTS_DIR" ]; then
     chmod -R 750 "$OMS_CERTS_DIR"
 fi
 
+%Postinstall_590
+# Remove version file of NPM if present
+NPM_VERSION_FILE=/var/opt/microsoft/omsagent/state/npm_version
```

Comment:
For a multi-homing scenario with more than one workspace configured, this does not check all the possible paths. Have you tested this with multiple workspaces?